<!-- Launch file for the rrt-detector and the assigner -->


<launch>
<arg name="robot_name" default="$(env ROBOT_NAME)"/>
<arg name="eta" value="1.0"/>
<arg name="Geta" value="15.0"/>

<group ns="$(arg robot_name)">
  <node pkg="rrt_exploration" type="global_rrt_detector" name="global_detector" output="screen">
  <param name="eta" value="$(arg Geta)"/>
  <param name="map_topic" value="/map"/>
  </node>
  
  <node pkg="rrt_exploration" type="local_rrt_detector" name="local_detector" output="screen">
  <param name="eta" value="$(arg eta)"/>
  <param name="map_topic" value="/map"/>
  <param name="robot_frame" value="/$(arg robot_name)/base_link"/>
  </node>
 
  
  <node pkg="rrt_exploration" type="filter.py" name="filter" output="screen">
  <param name="map_topic" value="/map"/>
  <param name="info_radius" value="25"/>
  <param name="costmap_clearing_threshold" value="50"/>
  <param name="goals_topic" value="/detected_points"/> 
  <param name="n_robots" value="1"/>
  <param name="rate" value="100"/>
  </node>


  

</group>
</launch>
